test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,12 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+        assertFalse(Utils.isAnyNull("abc", 1));
```

Comment:
You might want to add test cases to test for : 
1. Only null input
2. Empty input

Everything else looks good!